src/UI/Component/Symbol/Factory.php

```diff
@@ -62,19 +62,17 @@ public function icon(): Icon\Factory;
 	 * description:
 	 *   purpose: >
 	 *       Glyphs map a generally known concept or symbol to a specific concept in ILIAS.
-	 *       Glyphs are used when space is scarce.
+	 *       Glyphs SHOULD NOT be used with a text label. They are used when space is scarce.
```

Comment:
Do not state rules in the descriptions section. Please remove. If you are sure this rule is needed, move to the rules/usage section. Note that an additional rule would need a JF decision, so better remove in this PR.